### PR TITLE
feat(orgAdmins): show total user & team counts

### LINKED ISSUE #10393: Update user/team count views for Enterprise organization
[Full slack thread](https://parabol.slack.com/archives/C03NGEJUKCN/p1716321191476659)🔒 

Active user count isn't super relevant to our Enterprise customers as it fluctuates throughout the year and the sprint cycles vary. This means that an active user count might not be an accurate example of the need - and may impact decisions relating to the Enterprise tier required.

Now that users can see when a user last logged in on the Member view, a customer can export this list and determine their own definition of active (1 month/3 month etc).

For Enterprise organizations only, I'm suggesting:
* We remove the active user count in this view: https://action.parabol.co/me/organizations
* on the Plans & Billing view, we replace active user and active team counts with total users and total teams

### PR DESCRIPTION
**Pull request by tianrunhe**
# Description

Fixes #10393 

## Demo

Screenshot 2024-10-22 at 15 08 44

Screenshot 2024-10-30 at 08 54 59

## Testing scenarios

[Please list all the testing scenarios a reviewer has to check before approving the PR]

- [ ] Organizations page show total user count
  - Go to `http://localhost:3000/me/organizations`
  - See total number of users displayed

- [ ] Enterprise plan show total number of teams and users
  - Go to the billing page for an enterprise org: `http://localhost:3000/me/organizations/xxxxx/billing`
  - See total number of teams and users displayed above of plans

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label https://github.com/ParabolInc/parabol/labels/Skip%20Maintainer%20Review if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/modules/userDashboard/components/OrganizationRow/OrganizationRow.tsx [hunk 1]
```diff
@@ -139,9 +139,7 @@ const OrganizationRow = (props: Props) => {
               </StyledTagBlock>
             )}
           </RowInfoHeader>
-          <StyledRowInfoCopy>
-            {`${totalUsers} ${plural(totalUsers, 'User')} (${activeUserCount} Active)`}
-          </StyledRowInfoCopy>
+          <StyledRowInfoCopy>{`${totalUsers} ${plural(totalUsers, 'User')}`}</StyledRowInfoCopy>
```
Comment (nickoferrall) [inline packages/client/modules/userDashboard/components/OrganizationRow/OrganizationRow.tsx:142]: -1 we're removing `active` for all users here. It's still relevant for Team users, so I think it should only be hidden for Enterprise users

File: packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx [hunk 2]
```diff
@@ -131,6 +139,30 @@ const OrgPlans = (props: Props) => {
   return (
     <>
       <StyledPanel label='Plans'>
+        {billingTier === 'enterprise' && (
```
Comment (nickoferrall) [inline packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx:142]: +1 the plans section doesn't feel like the right place for this, to me. The plans explain what is offered in each tier, whereas this is information about usage. 

I'm happy to still approve the PR if that's where you and @charlesmathias what to put it, but I'd suggest checking with @ackernaut first

Comment (ackernaut) [reply packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx:142]: Sooner or later I’d like to create a usage section that is separate from the tiers and plans setion(s). Kind of like variants [we’ve explored here](https://www.figma.com/board/fdXXLw3wToZpVnpNvzoFl9/Org-Admin-%3E-Trial-Users?node-id=24-663&node-type=frame&t=YfZ6MBw2FTaFGWgl-11) 🔒 . It doesn’t have to happen in this PR.

I would like to see the usage section always at the top, regardless of how the other sections appear or are ordered below. If we need to add some special case banner to alert something at the very top, probably something that can be dismissed, we can always consider that later.

File: packages/client/modules/userDashboard/components/OrgBilling/OrgUsage.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  const {id: orgId, allTeamsCount, orgUserCount} = organization
+  const totalUserCount = orgUserCount.activeUserCount + orgUserCount.inactiveUserCount
+  const {history} = useRouter()
+
+  return (
+    <Panel className='mb-4 max-w-[976px]' label='Usage'>
+      <div className='flex items-center justify-around border-t border-solid border-slate-300 p-4 '>
+        <a
+          href=''
```
Comment (nickoferrall) [inline packages/client/modules/userDashboard/components/OrgBilling/OrgUsage.tsx:36]: +1 no href is included, so it shouldn't be an `a` tag here

File: packages/client/modules/userDashboard/components/OrgBilling/OrgUsage.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          }}
+          className='text-center text-sky-500 hover:text-sky-600'
+        >
+          <div className='mb-1 text-3xl font-bold'>{allTeamsCount}</div>
+          <div className='flex items-center justify-center text-base text-slate-600'>
+            Total teams
+          </div>
+        </a>
+        <a
+          href=''
```
Comment (nickoferrall) [inline packages/client/modules/userDashboard/components/OrgBilling/OrgUsage.tsx:49]: +1 same here

**Review by nickoferrall — APPROVED** [review]

**Pull request merged**